e2e/index.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@playwright/test";
+
+test("index page - displays available views and view links", async ({
+  page,
+}) => {
+  // 1. Navigate to the root of the application
+  await page.goto("/");
+
+  // 2. Wait for the page heading "Available Views" to become visible
+  await expect(
+    page.getByRole("heading", { name: /available views/i }),
+  ).toBeVisible();
+
+  // 3. Ensure at least one Alerts link is visible (guaranteed by other tests/config)
+  const alertsLink = page.getByRole("link", { name: /alerts/i }).first();
+  await alertsLink.waitFor({ state: "visible", timeout: 10000 });
+
+  // 4. (Optional) Verify the link's href leads to an alerts route pattern
+  const href = await alertsLink.getAttribute("href");
+  expect(href).toMatch(/^\/alerts\//);
+
+  // 5. Verify the logo image is displayed
+  const logoImage = page.locator("img[alt='Guardian Connector Explorer Logo']");
+  await expect(logoImage).toBeVisible();
+  expect(await logoImage.getAttribute("src")).toBe("/gcexplorer.png");
+});
+
+test("index page - language picker functionality", async ({ page }) => {
+  // 1. Navigate to the root of the application
+  await page.goto("/");
+
+  // 2. Wait for the language picker button to be visible
+  const languageButton = page
+    .locator("button")
+    .filter({ hasText: /English|Español|Nederlands|Português/i });
+  await languageButton.waitFor({ state: "visible", timeout: 10000 });
+
+  // 3. Verify the button shows current language
+  const buttonText = await languageButton.textContent();
+  expect(buttonText).toMatch(/English|Español|Nederlands|Português/);
```

Comment:
```suggestion
  expect(["English", "Español", "Nederlands", "Português"]).toContain(
    buttonText?.trim(),
  );
```

We might add more languages in the future. I don't think this test should fail if that's the case.

A similar change can be made above on line 35.